src/test/rustdoc-ui/issue-88756-default-output.rs

```diff
@@ -0,0 +1,2 @@
+// compile-flags: -h
```

Comment:
Hi @GuillaumeGomez !
I'm unable to figure out how to make the test for no f;gas. I'm not sure even if this `-h` is correct here as the key says `compile-flags` but it did give the same results as `rustdoc -h`. Could you please point me to any existing similar tests?

Reply:
That is actually a good question since the current file name is passed as argument... I can't find documentation about this either.

@jyn514: Do you know if there is a way to prevent any argument to be passed?

Reply:
You may need a run-make test instead of a UI test.

Reply:
Hi @jyn514 and @GuillaumeGomez !
I added the run-make test, could you please check if I did things the right way?